relay-server/src/services/processor.rs

```diff
@@ -1525,12 +1525,10 @@ impl EnvelopeProcessorService {
         dynamic_sampling::ensure_dsc(state);
         let filter_run = event::filter(state, &self.inner.global_config.current())?;
 
-        let mut sampling_should_drop = false;
-
         if self.inner.config.processing_enabled() || matches!(filter_run, FiltersStatus::Ok) {
             let sampling_result = dynamic_sampling::run(state, &self.inner.config);
             // Remember sampling decision, before it is reset in `dynamic_sampling::sample_envelope_items`.
-            sampling_should_drop = sampling_result.should_drop();
+            let sampling_should_drop = sampling_result.should_drop();
```

Comment:
`sampling_should_drop` is only used on line 1535 below. What do you think about inlining the function call there?